src/pages/home/HeaderView.js

```diff
@@ -146,6 +149,16 @@ const HeaderView = (props) => {
                                     </Text>
                                 )}
                             </View>
+                            {brickRoadIndicator === CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR && (
+                                <View style={[styles.alignItemsCenter, styles.justifyContentCenter]}>
+                                    <Icon
+                                        src={Expensicons.DotIndicator}
+                                        fill={colors.red}
+                                        height={variables.iconSizeSmall}
+                                        width={variables.iconSizeSmall}
+                                    />
+                                </View>
```

Comment:
NAB, I think it's probably time to make this a re-usable component.

Reply:
I would be happy to do that in a follow up.